### PR TITLE
Plt#10 active admin

### LINKED ISSUE #10: Set up ActiveAdmin
[Link to ActiveAdmin](https://github.com/activeadmin/activeadmin)

### PR DESCRIPTION
**Pull request by wonook**
Issue #10  - ActiveAdmin

1.Installed ActiveAdmin for the application
2.Integrated the current CanCan abilities to grant access to ActiveAdmin
3.Integrated models to display with ActiveAdmin
4.Filled up the dashboard to have a quick look on the application's elements

  - You can search, view, edit, delete elements of the application with ease without having to create separate views for each of the elements for the application.

  - You can have access to the ActiveAdmin interface under `localhost:3000/admin/` once you're logged in with an admin account.

Author : @wonook - Won Wook SONG (wsong0512@gmail.com)

This pull request closes #10 

---

Screenshots
Imgur
Imgur
Imgur
Imgur

### CODE REVIEW

**Comment by wonook** [comment]
I think it's done. Feedback please!

**Comment by wonook** [comment]
I think it's done. Feedback please!

File: config/initializers/active_admin.rb [hunk 1]
```diff
@@ -0,0 +1,233 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
```
Comment (azelma) [inline config/initializers/active_admin.rb]: Is there a reason for leaving in all this commented-out boilerplate?

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          Installation.unscoped.order("installation").map do |post|
+            li link_to(post.installation, admin_post_path(post))
+          end
+        end
+      end
+
+      column do
+        panel "Sites" do
+          Site.unscoped.order("installation_id").map do |site|
+            li link_to(site.name + " (" + Installation.find(site.installation_id).installation + ")", admin_post_path(site))
```
Comment (azelma) [inline app/admin/dashboard.rb]: Minor readability tweak, but this line is getting pretty long. Might do something like:
```
installation = Installation.find(site.installation_id).installation
li link_to("#{site.name} (#{installation})", admin_post_path(site))
```

File: config/initializers/active_admin.rb (hunk 1 above)
Comment (wonook) [inline config/initializers/active_admin.rb]: I thought I'd let other developers know the options that we could apply for `ActiveAdmin`. It was auto-generated, and I didn't find a reason to delete them. We have a similar case for `config/initializers/devise.rb`, and I sort of liked the way how I could see all of the options and configure them without having to search them up. All we have to do to apply or take out something is just to comment or uncomment particular lines.

File: app/admin/dashboard.rb (hunk 2 above)
Comment (wonook) [inline app/admin/dashboard.rb]: Thanks for the advice! I'll make that change.

**Comment by wonook** [comment]
If there are anything more to say, please mention it before 11AM GMT+0. 
If there are no more issues then, I'll merge this one in since we have a few issues waiting for this one to be merged.

**Pull request merged**